ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -124,6 +124,44 @@ objects:
 
         mkdir -p "${HOME}"
 
+        # Install python-openstackclient which is currently required to run
+        # some tests (in parallel mode). See: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/in_tree.go#L956
+        sudo bash <<EOF_SUDO
```

Comment:
You probably won’t have sudo

Reply:
The permissions issues are making `pip install --user ...` look more appealing ;)

Reply:
yeah, but like I mentioned on slack we don't have 'pip' either. I have to Yum install something.